digit-ui-esbuild/products/dashboard/src/styles/input.css

```
@@ -102,12 +102,82 @@
   }
 
+  .dashboard-widget-surface > header.dashboard-drag-handle,
+  .dashboard-widget-surface > header.dashboard-widget-header {
+    @apply tw-pt-4 tw-pb-0 tw-flex-col tw-items-stretch tw-justify-start;
+    /* Single source of title ↔ description spacing for charts + tables */
+    gap: 0.125rem;
+  }
+
+  .dashboard-widget-header-title-row {
+    @apply tw-min-w-0 tw-items-start;
+    padding-right: 1.75rem;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)